media-sound/ncmpcpp/ncmpcpp-0.9.2_p20240724.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+NCMPCPP_COMMIT="68daf44032784a5b5b74781a9bf3826053eff4a2"
+
+DESCRIPTION="Featureful ncurses based MPD client inspired by ncmpc"
+HOMEPAGE="https://rybczak.net/ncmpcpp/"
```

Comment:
Why was the GitHub upstream removed from HOMEPAGE?

Reply:
[Devmanual](https://devmanual.gentoo.org/ebuild-writing/variables/#ebuild-defined-variables) states that the code upstream might be listed if it's hard to find on the upstream homepage. And I haven't event found a single link to the GitHub repository on the upstream homepage. And if there is indeed a link, it is hidden pretty well.

Other than that, I personally like having the code upstream in my packages so I can easily click on the link when searching with eix, but that's just personal taste^^